api/search/queryprocessor.py

```diff
@@ -159,7 +159,7 @@ def _get_targets(self, path):
                     c: [copy.deepcopy(target)] for c in self.containers
                 }
             else:
-                return {path_parts[-min_length-1]: [target]}
+                return {path_parts[-2]: [target]}
```

Comment:
The implementation for search that I ported on master from st_prototype is slightly more complex (search on analyses was supported). `min_length` is used there, but not here. I just forgot to remove it...